docs-website/src/pages/docs/_components/QuickstartCards/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import clsx from "clsx";
+import styles from "./quickstartcards.module.scss";
+import QuickstartCard from '../QuickstartCard'
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import Link from "@docusaurus/Link";
+import {
+  ThunderboltTwoTone,
+  ApiTwoTone,
+  DeploymentUnitOutlined,
+  SyncOutlined,
+  CodeTwoTone,
+  QuestionCircleTwoTone,
+  SlidersTwoTone,
+  HeartTwoTone,
+} from "@ant-design/icons";
+
+const quickstartContent =  [
+{
+    title: "Quickstart with DataHub →",
+    icon: "datahub-logo-color-mark",
+    to: "docs/quickstart",
+    color: '#FFF',
+    fontColor: '#091013',
+  },
+{
+    title: "Learn about Managed DataHub →",
```

Comment:
can we make this a non-breaking space to avoid the arrow being on it's own line

<img width="868" alt="image" src="https://github.com/datahub-project/datahub/assets/12566801/2a084fc7-9a05-40e3-b9b8-68bf455e2fd0">